sites/cheerpj/src/content/docs/11-guides/basic-server-setup.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Basic Server Setup
+description: How do serve your own Java application with CheerpJ
+---
+
+## Overview
+
+To run your Java applications in the browser, you will need to self host the application files. This guide will cover some simple HTTP servers that can be used for basic testing and will discuss NGINX for a more advanced testing setup.
+
+> [!note] Important
+> The content of this guide is specifically made for testing setups and should not be used for a production environments. For production environments, it is essential to utilize HTTPS, for security reasons but also since it allows for a proper cross-origin setup. Localhost does not follow the same CORS (Cross-Origin Resource Sharing) policies, making it convenient for testing purposes.
```

Comment:
I am not sure about which CORS issues are you thinking about here. Can you elaborate?